pkg/install/v1alpha1/cstor_pool.go

```diff
@@ -197,6 +201,11 @@ spec:
       template:
         metadata:
           labels:
+            monitoring: pool_exporter_prometheus
```

Comment:
if this is internal to openebs than can we have `openebs.io/` as the prefix

Reply:
Saw simialr label for jiva target deployment and used the same here also.

Reply:
jiva was designed much earlier. It is good to have org name as namespace i.e. prefix
It help to categorically filter openebs specific resources.
It helps other tools to avoid modifying these labels & annotations as they are considered internal to openebs working.

Reply:
alright